bitnami/kubeapps/Chart.yaml

```diff
@@ -25,4 +25,4 @@ maintainers:
 name: kubeapps
 sources:
   - https://github.com/kubeapps/kubeapps
-version: 7.0.4
+version: 7.0.5
```

Comment:
Hi @dlaloue-vmware ,
Please have in mind adding new features to the Helm Chart would require bumping the minor version following semver.
```suggestion
version: 7.1.0
```

Reply:
for scope, we only changed how it is set to fix a potential duplicate issue.
for cookie-refresh, is it considered a feature or a bug fix?

I can do either way. this is an opportunity to learn what makes a change to be considered a patch or minor.

Reply:
I think providing new options in the values.yaml is changing/updating the "public API" of the chart, even though it was possible to set those options otherwise. According to [semver (point 7):](https://semver.org/)

> Minor version Y (x.Y.z | x > 0) MUST be incremented if new, backwards compatible functionality is introduced to the public API.

So +1 to a minor version change. And +1 for this PR from me :) Thanks!